mkdocs.yml

```
@@ -26,6 +26,7 @@ pages:
       - 'Telegram': 'connectors/telegram.md'
       - 'Rocket.Chat': 'connectors/rocketchat.md'
       - 'Matrix': 'connectors/matrix.md'
+      - 'WebexTeams': 'connectors/webexteams.md'
```

Let's put a space here too

```suggestion
      - 'Webex Teams': 'connectors/webexteams.md'
```